test/run_tests.pl

```diff
@@ -65,6 +65,16 @@ BEGIN
 my @alltests = find_matching_tests("*");
 my %tests = ();
 
+sub reorder {
+    my $key = $_;
+
+    # for parallel test runs, do slow tests first
+    if ($jobs >= 1 && $key =~ m/test_ssl_new|test_fuzz/) {
```

Comment:
`if $jobs > 1`, right?

Reply:
> `if $jobs > 1`, right?

Oops, yes.
For some experiments I had changed `>` to `>=` and forgot to restore that. Done.